{
  "warnings" : [ {
    "message" : "['$Main.java':10] on 'void Main::main(String*[] args)': [STATEMENT] POSSIBLE: uncaught runtime exception in main method"
  }, {
    "message" : "['Main.java':19:6] on 'void Main::main(String*[] args)': [EXPRESSION] DEFINITE: the assertion DOES NOT hold"
  } ],
  "files" : [ "report.json", "void_Main.main(String_[]_args).json" ],
  "info" : {
    "cfgs" : "14",
    "duration" : "99ms",
    "end" : "2025-09-10T17:56:07.487+02:00",
    "expressions" : "50",
    "files" : "1",
    "globals" : "7",
    "members" : "126",
    "programs" : "1",
    "start" : "2025-09-10T17:56:07.388+02:00",
    "statements" : "28",
    "units" : "75",
    "version" : "0.1",
    "warnings" : "2"
  },
  "configuration" : {
    "analysisGraphs" : "NONE",
    "descendingPhaseType" : "NONE",
    "dumpForcesUnwinding" : "false",
    "fixpointWorkingSet" : "OrderBasedWorkingSet",
    "glbThreshold" : "5",
    "hotspots" : "unset",
    "jsonOutput" : "true",
    "openCallPolicy" : "ReturnTopPolicy",
    "optimize" : "false",
    "recursionWideningThreshold" : "5",
    "semanticChecks" : "AssertChecker",
    "serializeInputs" : "false",
    "serializeResults" : "true",
    "shouldSmashError" : "unset",
    "syntacticChecks" : "",
    "useWideningPoints" : "true",
    "wideningThreshold" : "5",
    "workdir" : "java-outputs/svcomp/NullPointerException4"
  }
}